### DISCUSSION TITLE
Any advice on labels?

### DISCUSSION THREAD

**Discussion by lonelyadmin** [snipe/snipe-it#10155] [General]
Shopping for a label printer and was wondering if there are any recommendations for label types? UV fading and adhesive failing seem to be difficult qualities to measure.

**Comment by jonas-noa** [comment]
We are soon going to test two Brother printers of the P-series. One of them has labels suitable for outside. I let you know how it goes.

**Reply by lonelyadmin** [reply]
Thanks. I've been at places where we had to replace lables yearly....not fun.
